js/src/cli/integrations.ts

```diff
@@ -31,6 +31,25 @@ export default class ConnectionsCommand {
       console.log(chalk.red((error as any).message));
       return;
     }
+    const removeIntegrationId = options.remove || '';
```

Comment:
The empty string fallback for removeIntegrationId could lead to unexpected behavior. Consider throwing an error if the ID is empty or invalid format instead.